style/client.css

```diff
@@ -494,6 +495,7 @@ p.or:after {
 	border-color: #555555;
 	background: #FFFFFF;
 	background: -webkit-gradient(linear, left top, left bottom, from(#dce4ec), to(#c6d3e0));
+	background: -moz-linear-gradient(top,  #dce4ec, #c6d3e0);
```

Comment:
Are you telling me Firefox still doesn't support unprefixed `linear-gradient`? It's literally 2015. I find this hard to believe.